spec/services/auth_token_generator_service_spec.rb

```diff
@@ -1,11 +1,16 @@
 RSpec.describe AuthTokenGeneratorService do
   describe ".call" do
-    let(:subscriber) { create(:subscriber) }
     let(:secret) { Rails.application.secrets.email_alert_auth_token }
     let(:algorithim) { "HS256" }
     let(:expiry) { 1.day.from_now }
+    let(:data) do
+      {
+        token: "token",
```

Comment:
Minor: it's a bit confusing for the token data to be a `token`. Could we use a string like `"data"`?